### ISSUE TITLE
Layout for SAOS

### ISSUE THREAD

**Issue by pawelczak** [CeON/saos#3]
Prepare layout for web part of project SAOS. Layout should be modern, light, fast, easy to use and it must implement WCAG 2.0 rules.
